corehq/apps/reports/standard/monitoring.py

```diff
@@ -1428,9 +1428,11 @@ def _rows_by_user(self, report_data):
         last_form_by_user = self.es_last_submissions()
         for user in self.users_to_iterate:
             active_cases = int(report_data.active_cases_by_owner.get(user["user_id"].lower(), 0)) + \
-                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]])
+                sum([int(report_data.active_cases_by_owner.get(group_id, 0)) for group_id in user["group_ids"]]) + \
```

Comment:
Why don't you have to update https://github.com/dimagi/commcare-hq/blob/location-case-reports/corehq/apps/reports/standard/monitoring.py#L1375-L1378 similarly, to include both the users in each group and those users' locations?